src/nvim/undo.c

```diff
@@ -810,6 +814,12 @@ static bool serialize_header(bufinfo_T *bi, char_u *hash)
   return true;
 }
 
+/// Writes the undo header.
```

Comment:
maybe "an undo header" (as we except many undo headers, unlike _the_ undofile header)